src/include/kernel/abi.h

```diff
@@ -62,4 +62,7 @@
 /** \brief SOF ABI magic number "SOF\0". */
 #define SOF_ABI_MAGIC		0x00464F53
 
+/* SOF IPC4 ABI magic number "SOF4". */
+#define SOF_IPC4_ABI_MAGIC	0x34464F53
```

Comment:
Need to watch this as there may be some users who will ```printf(%s)``` this an rely on the NULL termination. Can you check for any code that prints abi.

Reply:
@lgirdwood Thanks for reminder. I will have a check.